src/main/java/com/backend/soullive_a/dto/response/ModelRecentWorkResponse.java

```diff
@@ -0,0 +1,21 @@
+package com.backend.soullive_a.dto.response;
+
+import com.backend.soullive_a.entity.model.Model;
+import jakarta.persistence.Column;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import lombok.Builder;
+
+@Builder
+public record ModelRecentWorkResponse(
+        String imageUrl,
+
+        Integer year,
+        String category,
+        String title,
+        String genre,
+        String role
```

Comment:
피그마랑 최대한 순서 맞춰서 써주세요
```suggestion
        String role,
        String genre
```